app/screens/interactive_dialog/dialog_element.js

```diff
@@ -133,8 +134,21 @@ export default class DialogElement extends PureComponent {
                     roundedBorders={false}
                 />
             );
+        } else if (type === 'radio') {
+            return (
+                <RadioSetting
+                    id={name}
+                    label={displayName}
+                    helpText={helpText}
+                    errorText={errorText}
+                    options={options}
+                    theme={theme}
+                    value={value || options[0].value}
```

Comment:
I see you are setting here the default, shouldn't it be the component default behaviour? otherwise I'd say to change this into a required prop to ensure functionality